studio/src/editors/variant-picker.js

```diff
@@ -9,23 +9,42 @@ const VARIANTS = [
     { label: 'Special offers', value: 'special-offers', surface: 'acom' },
 ];
 
-const renderVariants = () => {
-    return VARIANTS.map(
-        (variant) =>
-            html`<sp-menu-item value="${variant.value}"
-                >${variant.label}</sp-menu-item
-            >`,
-    );
-};
-
-export const renderVariantPicker = (value, handler, id) => {
-    return html`<sp-picker
-        id="${id || 'card-variant'}"
-        label="Card Variant"
-        size="m"
-        value=${value}
-        @change="${handler}"
-    >
-        ${renderVariants()}
-    </sp-picker>`;
-};
+class VariantPicker extends LitElement {
+    static properties = {
+        initvalue: { type: String },
+    };
+
+    get value() {
+        return this.querySelector('sp-picker')?.value;
+    }
+
+    createRenderRoot() {
+        return this;
+    }
+
+    renderVariants = () => {
+        return VARIANTS.map(
+            (variant) =>
+                html`<sp-menu-item value="${variant.value}"
+                    >${variant.label}</sp-menu-item
+                >`,
+        );
+    };
+
+    handler = () => {
+        this.dispatchEvent(new CustomEvent('selectionchange'));
+    };
+
+    render() {
+        return html`<sp-picker
+            label="Card Variant"
+            size="m"
+            value=${this.value || this.initvalue}
```

Comment:
`this.initvalue` sounds like a method name,  how about `initialValue` or `defaultValue`?